_data/setup-scala.yml

```
@@ -1,6 +1,6 @@
 linux-x86-64: curl -fL https://github.com/coursier/coursier/releases/latest/download/cs-x86_64-pc-linux.gz | gzip -d > cs && chmod +x cs && ./cs setup
 linux-arm64: curl -fL https://github.com/VirtusLab/coursier-m1/releases/latest/download/cs-aarch64-pc-linux.gz | gzip -d > cs && chmod +x cs && ./cs setup
```

```suggestion
linux-arm64: curl -fL https://github.com/coursier/launchers/raw/master/cs-aarch64-pc-linux.gz | gzip -d > cs && chmod +x cs && ./cs setup
```
Now, we could do this, which at the point of typing this comment points to coursier 2.1.25-M16... as the new Linux aarch64 launchers are still locked behind milestone versions of 2.1.25, so they aren't available behind `latest`... but perhaps it's better to wait for stable?

`VirtusLab/coursier-m1` is otherwise unmaintained.

Either way, I opt to leave it like this for the time being, perhaps until 2.1.25 is fully out?
cc @alexarchambault